### ISSUE TITLE
β in the first example have three values

### ISSUE THREAD

**Issue by St-ZFeng** [jmxpearson/rlfit#3]
![image](https://github.com/user-attachments/assets/01d739a9-ae60-48a9-8804-deb55931fdb0)
Thank you for such a convenient code for fitting the model.

When I run the example you provided for a single learning rate, I find that three values appear for beta. This is confusing to me. If the first value is the softmax parameter, what are the next two parameters? You wrote:

```
% model has only one parameter, the learning rate, between 0 and 1
% in general, there will be one of these for each parameter (excluding the
% softmax parameter)
```
It seems like there should be two values here.

**Comment by jmxpearson** [comment]
The additional parameter is a perseveration bonus. See [here](https://github.com/jmxpearson/rlfit/blob/master/example.m#L18).
